packages/plugins/test/mocha/cli/deploy.spec.ts

```diff
@@ -404,12 +404,11 @@ describe('Deploy CLI command', () => {
       expect(existsSync(deploymentArtifactDirPath)).to.be.false
     })
 
-    // This test checks that Foundry's simulation can fail after the transactions have been
-    // collected, but before any transactions are broadcasted. This is worthwhile to test because
-    // the `SphinxModule` doesn't revert if a user's transactions causes the deployment to be marked
-    // as `FAILED`. If the Foundry plugin doesn't revert either, then Foundry will attempt to
-    // broadcast the deployment, which is not desirable.
-    it('Reverts if the deployment fails during the simulation', async () => {
+    // This test checks that an error is thrown if a transaction fails during execution. This is worthwhile
+    // to test because the `SphinxModule` doesn't revert if a user's transactions causes the deployment to be
+    // marked as `FAILED`. If the plugin doesn't revert either, then the user may believe their deployment
+    // completed successfully.
+    it('Errors if the deployment fails during execution', async () => {
```

Comment:
The previous version of this test relied on the simulation running against local nodes. Since we do expect an error to be thrown during execution if an action reverts when the transactions are being broadcast, I updated this test to reflect that behavior.